config/settings.yml

```diff
@@ -2,4 +2,4 @@ redis:
   hostname: localhost
   port: 6379
 
-secret_token: not_a_sekret!
\ No newline at end of file
+secret_token: 6QeudURqMZHfC7w2hh68Q2SUEtLPy9ApunMeZbVZQrR4Vyc6YPAvjDPkFjYUpees
```

Comment:
This ok?

Reply:
It's not awesome, but also probably not the worst.

Reply:
robot-console is pretty well restricted by requiring users to be on VPN and in the right workgroup.  and i don't think it really has any login functionality or other signed cookie authN/authZ type stuff.  so probably not much an attacker could do even if they could forge cookies?

if we did still feel icky about it, we could stick the token in vault and override the setting via env var, as we do for some other things that really do need to remain secret.